src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -333,19 +333,17 @@ function ($match) use ($ret) {
 
     private function mostRecentlyEnteredValue($entered)
     {
-        $tempEntered = $entered;
-
         // Determine the most recent value that the user entered
-        if (false !== strpos($entered, ',')) {
-            $choices = explode(',', $entered);
-            $lastChoice = trim($choices[\count($choices) - 1]);
+        if (false === strpos($entered, ',')) {
+            return $entered;
+        }
 
-            if (\strlen($lastChoice) > 0) {
-                $tempEntered = $lastChoice;
-            }
+        $choices = explode(',', $entered);
+        if (\strlen($lastChoice = trim($choices[\count($choices) - 1])) > 0) {
+            return $lastChoice;
         }
 
-        return $tempEntered;
+        return $entered;
```

Comment:
this method was added as part of a bugfix not released yet, should we apply this change on 3.4?

Reply:
I will backport once tests pass :)